src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java

```diff
@@ -67,7 +64,25 @@ public void run(final UserInput userInput) {
         } else if (userInput.getGoal() == Goal.RELEASE) {
             reports.addAll(this.releaseUseCase.release(repository, platformNames));
         }
-        writeReportToDisk(userInput, platformNames, reports);
+        sendResponse(createResponse(reports, userInput, platformNames));
+    }
+
+    private void sendResponse(final ReleaseDroidResponse response) {
```

Comment:
I find this name a bit unclear. Maybe we can rename it as `processResponse` or something more meaningful?